docs/js/files/modules_FormModel.js.html

```diff
@@ -161,6 +161,12 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
     mapping: undefined,
 
     /**
+     * @property models
+     * @type Object
+     **/
+    models: undefined,
```

Comment:
I'm fine the way this is for now. Just lots of ways to setup a form model. It's a little concerning to make sure we've covered what the expected behavior is for all the ways. You can set mapping OR models OR both on the class level. At run time, you can set new mappings OR new mappings with new models OR just new models.